CONTRIBUTING.md

```diff
@@ -1,155 +0,0 @@
-# Contributing to Chef
```

Comment:
That isn't what we've been doing across our other repositories. If we want to do that we should probably keep it consistent across the other projects as well (such as chef-dk, omnibus, ohai and many others).